src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -120,5 +123,26 @@ public Task RavenRoot()
             HttpContext.Response.StatusCode = 301;
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/c-sharp-index-definition", "GET")]
```

Comment:
@IdanHaim The intention is to have Studio related endpoints under /studio/** (or sth similar), not only to move them to the StudioHandler